### ISSUE TITLE
If no datas, no rows in database, the mock is not launched

### ISSUE THREAD

**Issue by hrenaud** [NovaGaia/strapi-plugin-mock-datas#6]
Try to fix here:

```javascript
'use strict';

const _ = require('lodash');
const { getPluginService } = require('../utils/getPluginService');

const transform = async (strapi, ctx, next, datas) => {
  const tempConfigStore = await getPluginService('datasMockerServices').getConfigStore();

  if (tempConfigStore) {
    datas.configStore = tempConfigStore;
  } else {
    // add default value if settings has never been set.
    datas.configStore = { mockEnabled: false, addImageInRichtextEnabled: false };
  }

  await next();

  // ensure body exists, occurs on non existent route
  if (!ctx.body) {
    return;
  }

  // ensure no error returned.
  if (!ctx.body.data) { // <-- here ?
    return;
  }

  // execute response transforms
  getPluginService('transformServices').transformResponse(ctx, datas);
};

module.exports = { transform };

```

**Comment by hrenaud** [comment]
`ctx.body.data` is mandatory in `./server/services/transform.service.js` so it's not possible to bypass.

The plugin need one row in bdd :'(

**Comment by hrenaud** [comment]
"fixed" in v1.4.0.

When no data was saved, you now received mocked data, but not the custom field added in controllers. Warn is sent in console.
